tools/claude/plugins/dxos/hooks/track.sh

```
@@ -68,48 +68,48 @@ emit() {
 }
 
 emit_info() {
-  emit 'TASK-PLANNING PROJECT DIRECTIVE: `/dxos-project:project` (bare) — report the CURRENT project, do NOT list the registry. Resolve it: prefer the entry whose name matches this branch (strip a leading `claude/` and a trailing `-<hash>` from `git branch --show-current`); otherwise the single `active` entry for the current user (`whoami`). If neither resolves, say so and suggest `/dxos-project:project list`. Then report, compactly: (1) worktree directory + branch; (2) the project name, status, and its `tasks`/`design` paths and `prs`; (3) tracked-file state — `git status --short` and whether the branch is ahead of origin, naming every uncommitted file rather than summarising. Render EVERY file path as a repo-relative markdown link so it is clickable. (4) the next action, taken from the entry`s `resume` field and condensed to one line. No numbered options unless something genuinely needs a decision.'
+  emit 'TASK-PLANNING PROJECT DIRECTIVE: `/dxos:project` (bare) — report the CURRENT project, do NOT list the registry. Resolve it: prefer the entry whose name matches this branch (strip a leading `claude/` and a trailing `-<hash>` from `git branch --show-current`); otherwise the single `active` entry for the current user (`whoami`). If neither resolves, say so and suggest `/dxos:project list`. Then report, compactly: (1) worktree directory + branch; (2) the project name, status, and its `tasks`/`design` paths and `prs`; (3) tracked-file state — `git status --short` and whether the branch is ahead of origin, naming every uncommitted file rather than summarising. Render EVERY file path as a repo-relative markdown link so it is clickable. (4) the next action, taken from the entry`s `resume` field and condensed to one line. No numbered options unless something genuinely needs a decision.'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)